wagtailio/project_styleguide/templates/patterns/pages/showcase_page/showcase_page_fragment.html

```diff
@@ -0,0 +1,99 @@
+                                    alt="{% firstof item.alt_text item.image.alt %}"/>
+                            {% image item.logo max-120x100 as image_png %}
+                            <source srcset="{{ image_png.url }}"
+                                    type="image/png"
+                                    alt="{% firstof item.alt_text item.image.alt %}"/>
+                            {% image item.logo max-120x100 as base_img %}
+                            <img
+                                src="{{ base_img.url }}"
+                                height="{{ base_img.height }}"
+                                width="{{ base_img.width }}"
+                                class="showcase-card__logo"
+                                alt="{% firstof item.alt_text item.image.alt %}"/>
+                        </picture>
+                    {% endif %}
+                {% endif %}
+                <div class="showcase-card__content">
+                    <h2 class="showcase-card__heading">{{ item.title }}</h2>
+                    <p class="showcase-card__description">{{ item.description }}</p>
+                    <a class="showcase-card__link" href="{% if item.external_link %}{{ item.external_link }}{% else %}{% pageurl item.internal_link %}{% endif %}">{{ item.link_text }} <svg class="showcase-card__svg" width="15" height="16" aria-hidden="true">
+                        <use xlink:href="#{% if self.external_link %}open-link{% else %}arrow{% endif %}"></use>
+                    </svg></a>
+                </div>
+            </div>
+        {% endfor %}
+    </div>
+    <div>
+        <div
+            hx-boost="true"
+            class="pagination pagination--showcase">
+            {% if showcase_items.has_previous %}
+                <a hx-get="{% pageurl page %}"
+                   hx-trigger="click"
+                   hx-target="#showcase_results"
+                   hx-push-url="true"
+                   hx-swap="innerHTML"
+                   href="?page={{ showcase_items.previous_page_number }}{% if current_sector %}&sector={{ current_sector }}{% endif %}" class="pagination__previous button button--reverse">
+                    <p class="button__text">
+                        Previous
+                        <span class="u-sr-only">- Go to page {{ showcase_items.previous_page_number}}</span>
+                    </p>
+                    <svg class="arrow" aria-hidden="true"><use xlink:href="#arrow"></use></svg>
+                </a>
+            {% endif %}
+
+            <span class="pagination__page-count mini-meta">
+                Page {{ showcase_items.number }} of {{ showcase_items.paginator.num_pages }}
+            </span>
+
+            {% if showcase_items.has_next %}
+                <a
+                    hx-trigger="click"
+                    hx-target="#showcase_results"
+                    hx-push-url="true"
+                    hx-swap="outerHTML"
+                    hx-swap="innerHTML"
```

Comment:
You have `hx-swap` twice and no `hx-get` here

Reply:
Good spot for the duplication, we don't need hx-get as above it we have hx-boost! :)